CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [5.7.2] - 2022-09-21
```

Comment:
I think this should 5.8.0, not 5.7.2. I would say this is a minor version, not a hotfix.

Reply:
Agreed, that's just me on autopilot 😅